service-users-server/src/main/java/io/pestakit/users/repositories/UserRepository.java

```diff
@@ -0,0 +1,7 @@
+package io.pestakit.users.repositories;
+
+import io.pestakit.users.entities.UserEntity;
+import org.springframework.data.repository.CrudRepository;
+
+public interface UserRepository extends CrudRepository<UserEntity, Long> {
+}
```

Comment:
add a findByUsername and a findByEmail if you want to lookup at creation time (previous comment)

Reply:
done in a5e2ffa13109f91eb4099836bce4c439e68b0044